installation/sources/build-and-install.md

```diff
@@ -1,223 +1,243 @@
-# Build and Install
+# Build and install
 
-[Fluent Bit](http://fluentbit.io) uses [CMake](http://cmake.org) as its build system. The suggested procedure to prepare the build system consists of the following steps:
+[Fluent Bit](http://fluentbit.io) uses [CMake](http://cmake.org) as its build system.
 
 ## Requirements
 
-- CMake >= 3.12
+- CMake 3.12 or greater. You might need to use `cmake3` instead of `cmake`.
 - Flex
-- Bison >= 3
+- Bison 3 or greater
 - YAML headers
 - OpenSSL headers
 
 ## Prepare environment
 
-> In the following steps you can find exact commands to build and install the project with the default options. If you already know how CMake works you can skip this part and look at the build options available. Note that Fluent Bit requires CMake 3.x. You may need to use `cmake3` instead of `cmake` to complete the following steps on your system.
-
-Change to the _build/_ directory inside the Fluent Bit sources:
-
-```bash
-$ cd build/
-```
-
-Let [CMake](http://cmake.org) configure the project specifying where the root path is located:
-
-```bash
-$ cmake ../
--- The C compiler identification is GNU 4.9.2
--- Check for working C compiler: /usr/bin/cc
--- Check for working C compiler: /usr/bin/cc -- works
--- Detecting C compiler ABI info
--- Detecting C compiler ABI info - done
--- The CXX compiler identification is GNU 4.9.2
--- Check for working CXX compiler: /usr/bin/c++
--- Check for working CXX compiler: /usr/bin/c++ -- works
-...
--- Could NOT find Doxygen (missing:  DOXYGEN_EXECUTABLE)
--- Looking for accept4
--- Looking for accept4 - not found
--- Configuring done
--- Generating done
--- Build files have been written to: /home/edsiper/coding/fluent-bit/build
-```
-
-Now you are ready to start the compilation process through the simple _make_ command:
+If you already know how CMake works you can skip this part and look at the build
+options available.
```

Comment:
```suggestion
If you already know how CMake works, you can skip this section and review the
available [build options](#general-options).
```

Reply:
Suggest adding a nice little cross link.

Reply:
I'm going to worry about crosslinks in a later iteration :D